### ISSUE TITLE
Installing from PyPI fails

### ISSUE THREAD

**Issue by beenje** [PierreRust/apigpio#8]
```
# pip install apigpio
Collecting apigpio
  Downloading apigpio-0.0.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-lrcpnl9o/apigpio/setup.py", line 4, in <module>
        with open('README.md', 'r') as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'README.md'
```

You should add a MANIFEST.in to be sure the file README.md is included in the package.

Note that PyPI doesn't render markdown. You might want to use a README.rst instead. There are some workarounds with pandoc otherwise: https://stackoverflow.com/questions/26737222/pypi-description-markdown-doesnt-work

**Comment by PierreRust** [comment]
I thought that with the setup.cfg, a markdown README would work (read it somewhere ...)
But you're right it does not work, it's probably easier and better to simply switch to rst.

**Comment by PierreRust** [comment]
I've fixed it in the new 0.0.2 version, should be working now.

**Comment by beenje** [comment]
Thanks for the quick fix!
